.github/README.md

```diff
@@ -192,20 +212,10 @@ if ($payment->wasPaid()) {
 It's basically the same as second step (customer return), it's triggered everytime the payment has changed, for example when the state of payment has been changed.
 
```

Comment:
Wanted to add link in here to the example "how to make new TheClient", just like this:
```md
> [see how to make TheClient](./README.md#theclient-instance)
```
> [see how to make TheClient](./README.md#theclient-instance)